frontend/src/index.jsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { BrowserRouter as Router, Route, Switch } from 'react-router-dom';
+import Login from './components/Login/Login';
+import Dashboard from './components/Dashboard/Dashboard';
+import './index.css';
+
+const App = () => {
+  return (
+    <Router>
+      <Switch>
+        <Route exact path="/">
```

Comment:
I would actually keep the index route for the dashboard and move the Login to `/login` route.

Also, consider adding Authenticated (or similar) to secure private routes and handle redirects to the login screen when access token is invalid.

```tsx
      <Switch>
        <Route exact path="/">
          <Login exact path="/login" />
        </Route>
        <Route path="/">
           <Authenticated>
              <Dashboard />
            <Authenticated/>
        </Route>
      </Switch>
```

Reply:
I have a plan in my head for how to implement this, but I think it makes more sense to do it when I add session management on the server as well. Ok if I punt it to my next PR?